<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VSAMainWindow</class>
 <widget class="QMainWindow" name="VSAMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>772</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Vision Soil Analyzer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="vsa_resources.qrc">
    <normaloff>:/Icons/Icons/IHClogo.png</normaloff>:/Icons/Icons/IHClogo.png</iconset>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_Particle">
        <item>
         <widget class="QParticleDisplay" name="widget_ParticleDisplay" native="true">
          <property name="minimumSize">
           <size>
            <width>600</width>
            <height>318</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>600</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QParticleSelector" name="widget_ParticleSelector" native="true">
          <property name="minimumSize">
           <size>
            <width>600</width>
            <height>330</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>600</width>
            <height>330</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_Sample">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCustomPlot" name="Qplot_PSD" native="true">
            <property name="minimumSize">
             <size>
              <width>383</width>
              <height>210</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QCustomPlot" name="QPlot_Roudness" native="true">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>210</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCustomPlot" name="QPlot_Angularity" native="true">
              <property name="minimumSize">
               <size>
                <width>241</width>
                <height>210</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewSample"/>
   <addaction name="actionSaveSample"/>
   <addaction name="actionLoadSample"/>
   <addaction name="separator"/>
   <addaction name="actionAnalyzeSample"/>
   <addaction name="actionUseLearning"/>
   <addaction name="actionAutomatic_Shape_Pediction"/>
   <addaction name="actionCalibrate"/>
   <addaction name="separator"/>
   <addaction name="actionSetLocation"/>
   <addaction name="separator"/>
   <addaction name="actionReport_Generator"/>
   <addaction name="actionSendResults"/>
   <addaction name="actionPrintResults"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
   <addaction name="actionNeuralNet"/>
   <addaction name="separator"/>
   <addaction name="actionHelp"/>
   <addaction name="actionAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAnalyzeSample">
   <property name="icon">
    <iconset resource="vsa_resources.qrc">
     <normaloff>:/Icons/Icons/Analyze-icon.png</normaloff>:/Icons/Icons/Analyze-icon.png</iconset>
   </property>
   <property name="text">
    <string>AnalyzeSample</string>
   </property>
  </action>
  <action name="actionNewSample">
   <property name="icon">
    <iconset resource="vsa_resources.qrc">
     <normaloff>:/Icons/Icons/New-icon.png</normaloff>:/Icons/Icons/New-icon.png</iconset>
   </property>
   <property name="text">
    <string>NewSample</string>
   </property>
  </action>
  <action name="actionLoadSample">
   <property name="icon">
    <iconset resource="vsa_resources.qrc">
     <normaloff>:/Icons/Icons/Open.png</normaloff>:/Icons/Icons/Open.png</iconset>
   </property>
   <property name="text">
    <string>LoadSample</string>
   </property>
  </action>
  <action name="actionSaveSample">
   <property name="icon">
    <iconset resource="vsa_resources.qrc">
     <normaloff>:/Icons/Icons/Save-icon.png</normaloff>:/Icons/Icons/Save-icon.png</iconset>
   </property>
   <property name="text">
    <string>SaveSample</string>
   </property>
  </action>
  <action name="actionSetLocation">
   <property name="icon">
    <iconset resource="vsa_resources.qrc">
     <normaloff>:/Icons/Icons/SetLocation-icon.png</normaloff>:/Icons/Icons/SetLocation-icon.png</iconset>
   </property>
   <property name="text">
    <string>SetLocation</string>
   </property>
  </action>
  <action name="actionSendResults">
   <property name="icon">
    <iconset resource="vsa_resources.qrc">
     <normaloff>:/Icons/Icons/SendSample-icon.png</normaloff>:/Icons/Icons/SendSample-icon.png</iconset>
   </property>
   <property name="text">
    <string>SendResults</string>
   </property>
  </action>
  <action name="actionPrintResults">
   <property name="icon">
    <iconset resource="vsa_resources.qrc">
     <normaloff>:/Icons/Icons/PrintResults-icon.png</normaloff>:/Icons/Icons/PrintResults-icon.png</iconset>
   </property>
   <property name="text">
    <string>PrintResults</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="vsa_resources.qrc">
     <normaloff>:/Icons/Icons/Help-icon.png</normaloff>:/Icons/Icons/Help-icon.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="vsa_resources.qrc">
     <normaloff>:/Icons/Icons/IHClogo.png</normaloff>:/Icons/Icons/IHClogo.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="vsa_resources.qrc">
     <normaloff>:/Icons/Icons/Settings-icon.png</normaloff>:/Icons/Icons/Settings-icon.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionNeuralNet">
   <property name="icon">
    <iconset resource="vsa_resources.qrc">
     <normaloff>:/Icons/Icons/NeuralNetwork.PNG</normaloff>:/Icons/Icons/NeuralNetwork.PNG</iconset>
   </property>
   <property name="text">
    <string>NeuralNet</string>
   </property>
  </action>
  <action name="actionUseLearning">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="vsa_resources.qrc">
     <normaloff>:/Icons/Icons/Study_fade.png</normaloff>
     <normalon>:/Icons/Icons/Study.png</normalon>:/Icons/Icons/Study_fade.png</iconset>
   </property>
   <property name="text">
    <string>useLearning</string>
   </property>
  </action>
  <action name="actionCalibrate">
   <property name="icon">
    <iconset resource="vsa_resources.qrc">
     <normaloff>:/Icons/Icons/1cent.jpg</normaloff>:/Icons/Icons/1cent.jpg</iconset>
   </property>
   <property name="text">
    <string>Calibrate</string>
   </property>
  </action>
  <action name="actionAutomatic_Shape_Pediction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="vsa_resources.qrc">
     <normaloff>:/Icons/Icons/NeuralNet.png</normaloff>
     <normalon>:/Icons/Icons/NeuralNet_.png</normalon>:/Icons/Icons/NeuralNet.png</iconset>
   </property>
   <property name="text">
    <string>Automatic Shape Pediction</string>
   </property>
  </action>
  <action name="actionReport_Generator">
   <property name="icon">
    <iconset resource="vsa_resources.qrc">
     <normaloff>:/Icons/Icons/Report.png</normaloff>:/Icons/Icons/Report.png</iconset>
   </property>
   <property name="text">
    <string>Report Generator</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QParticleSelector</class>
   <extends>QWidget</extends>
   <header>qparticleselector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QParticleDisplay</class>
   <extends>QWidget</extends>
   <header>qparticledisplay.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="vsa_resources.qrc"/>
 </resources>
 <connections/>
</ui>
